clients/python/test_python_client.py

```diff
@@ -63,68 +65,77 @@
 # or AIRFLOW__CORE__LOAD_EXAMPLES environment variable set to True
 DAG_ID = "example_bash_operator"
 
-# Enter a context with an instance of the API client
-with airflow_client.client.ApiClient(configuration) as api_client:
-    errors = False
 
-    print("[blue]Getting DAG list")
-    dag_api_instance = dag_api.DAGApi(api_client)
-    try:
-        api_response = dag_api_instance.get_dags()
-        print(api_response)
-    except airflow_client.client.OpenApiException as e:
-        print(f"[red]Exception when calling DagAPI->get_dags: {e}\n")
-        errors = True
-    else:
-        print("[green]Getting DAG list successful")
+# Enter a context with an instance of the API client
```

Comment:
I like this one is converted to pytest :) :+1:

Reply:
However there is one problem (small) with that. We are currently using `test_python_client.py` as an executable  script to run tests manually when verifying the client. 

See https://github.com/apache/airflow/blob/main/dev/README_RELEASE_PYTHON_CLIENT.md#verify-the-release-candidate-by-contributors

So we should either instruct the users to use `pytest` to run those tests, or turn it into runnable script (i.e. add 

```
if __name__ == "__main__":
    test_python_client()
```

I think the second option is best.

Reply:
Yes agree, add this one :)